### PR TITLE
Add fed job

### PR DESCRIPTION
**Pull request by lukefrake**

### CODE REVIEW

File: _posts/2016-02-16-looking-for-a-front-end-developer.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* Experience working in an agile team
+* Experience working with clients and overall great communication skills
+
+## What we give you
+
+* Competitive Salary - D.O.E
+* Your own MacBook Pro
+* Flexible working hours
+* Home working
+* Freedom to get shit done!
```
Comment (rafriki) [inline _posts/2016-02-16-looking-for-a-front-end-developer.md]: :'(

File: _posts/2016-02-16-looking-for-a-front-end-developer.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+banner:         "/assets/images/blog/question.jpg"
+---
+
+We're getting busy so we're looking for another great person to join our team.
+
+## What we're looking for
+
+We believe that to achieve brilliance, you need a brilliant team. We're looking for a front-end developer who's ready to grow with our business. Working closely with every member of the Space Between team, this role comes with room to learn and teach.
+
+Our current tech bread and butter consists of; Angular.JS, Azure, AWS, .NET, Umbraco, Git not forgetting JavaScript, HTML & CSS.
```
Comment (rafriki) [inline _posts/2016-02-16-looking-for-a-front-end-developer.md]: ; wrong

File: _posts/2016-02-16-looking-for-a-ui-designer.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+* Experience working in an agile team
+* Experience working with clients and overall great communication skills
+
+## What we give you
+
+* Competitive Salary - D.O.E
+* Your own MacBook Pro
+* Flexible working hours
+* Home working
+* Freedom to get shit done!
```
Comment (rafriki) [inline _posts/2016-02-16-looking-for-a-ui-designer.md]: :/

File: _posts/2016-02-16-looking-for-a-front-end-developer.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+layout:         post
+title:          "We want a front-end developer"
```
Comment (rafriki) [inline _posts/2016-02-16-looking-for-a-front-end-developer.md]: ?

File: _posts/2016-02-16-looking-for-a-front-end-developer.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+description:    "We're looking for a front-end developer"
+categories:     space between
+banner:         "/assets/images/blog/question.jpg"
+---
+
+We're getting busy so we're looking for another great person to join our team.
+
+## What we're looking for
+
+We believe that to achieve brilliance, you need a brilliant team. We're looking for a front-end developer who's ready to grow with our business. Working closely with every member of the Space Between team, this role comes with room to learn and teach.
```
Comment (rafriki) [inline _posts/2016-02-16-looking-for-a-front-end-developer.md:15]: teach who?

Comment (lukefrake) [inline _posts/2016-02-16-looking-for-a-front-end-developer.md:15]: Us

**Comment by rafriki** [comment]
:+1:

**Pull request merged**